docs/README.md

```diff
@@ -207,12 +208,19 @@ kubectl create secret generic \
   registry-secret --from-file=$HOME/.docker/config.json 
 ```
 
-Create of-builder and of-buildkit:
+Create of-builder, of-buildkit and network policies:
 
 ```
 kubectl apply -f ./yaml
 ```
 
+Add `openfaas-system` to namespace where your Ingress Controller lives. For example if Ingress Controller is deployed in `ingress-nginx` namespace:
```

Comment:
Grammar still not quite right on this sentence.

if Ingress Controller should be if <the> or if <a>.

IngressController should be one word.

Add openfaas-system should really be "Add a role of "openfaas-system" using a label.

Same again with "to namespace" - this should be "to the namespace" 

Controllers don't live, they are deployed. "Where you deployed it" for instance